### DISCUSSION TITLE
Project Continuity is still active?

### DISCUSSION THREAD

**Discussion by rafapcarvalho** [hyperjumptech/grule-rule-engine#448] [General]
Dear Team,

I am writing to inquire about the current status of the Grule Rule Engine project.

We are using the Grule Rule Engine in one of our company's projects and would like to know if the project is still active and if there are any plans for future updates and maintenance. Additionally, we are interested in using the Grule Rule Engine in many other projects in the future. If not, we will need to consider migrating to another rule engine.

We appreciate your attention to this matter and look forward to your response.

Best regards,
Rafael

**Comment by cloudvant** [comment]
I don't see any recent activity from the maintainer.

**Comment by ralphg6** [comment]
We started a fork to keep it alive.. Please help us to do it...

https://github.com/DataWiseHQ/grule-rule-engine

Raphael Pinto


Em dom., 9 de mar. de 2025 às 13:20, Victor Martin ***@***.***>
escreveu:
